src/autopas/cells/FullParticleCell.h

```diff
@@ -50,6 +50,17 @@ class FullParticleCell : public ParticleCell<Particle> {
 
   void addParticle(const Particle &p) override {
     std::lock_guard<AutoPasLock> guard(this->_cellLock);
+
+    // sanity check that ensures that only particles of the cells OwnershipState can be added. Note: is a cell is a
```

Comment:
```suggestion
    // sanity check that ensures that only particles of the cells OwnershipState can be added. Note: if a cell is a
```

Reply:
done in[37e2621](https://github.com/AutoPas/AutoPas/pull/782/commits/37e26218410380f7bc586d66bd83b316d23a3d28)